Gx3 3885 ⚡️ tools qa elements text box fill form and submit

E-Gregorio opened the pull request:
Summary

Se ejecutaron un total de 8 TCs,  automatizados en el entorno de Chromium. Todos los TCs fueron diseñados para validar diferentes escenarios de ingreso de datos en un formulario web en el sitio  demoqa.com/text-box.  Los resultados de las pruebas muestran que todas fueron exitosas, confirmando que el formulario se comporta correctamente en base a los  casos evaluados cumpliendo asi de esta manera con lo espeficado en la documentacion del proyecto.

Resumen de la Ejecusion de las preubas:

Total de Pruebas Ejecutadas: 8
Navegador Utilizado: Chromium
Tiempo de Ejecución Total: 46.47 segundos
Resultado General: Todas las pruebas fueron exitosas (✅ ALL TESTS PASSED).

Results:
![image](https://github.com/upex-galaxy/playwright-blackhole/assets/155954783/d9b0beb6-28a5-49f6-b2c9-9e63ac382f16)

Observations:
Ninguna

Commit:
GX3-3770: Actualizacion de la Nueva Rama y actualizacion del archivo sanity-gregorio.yml

Commit:
Agregados y probados casos de prueba TC1, TC2 y TC de la US GX3-3823 de ToolsQA Elements Buttons

Commit:
Actualizacion de Datos

Commit:
En el archivo sanity-gregorio.yml, actualicé los datos para integrar con Jira. Además, creé y ejecuté 8 pruebas, todas las cuales pasaron con éxito

Commit:
Actualizacion

Commit:
Actualizacion

saiotest committed:
Delete tests/test-1.spec.ts

saiotest committed:
Delete tests/test-2.spec.ts

saiotest commented on tests/specs/elements/TextBox_GX3-3885.test.ts:
```diff
@@ -0,0 +1,177 @@
+	test('TC13 Validar ingresar datos válidos en el formulario con "Current Address" y "Permanent Address" vacíos', async ({ page }) => {
+		await page.getByPlaceholder('Full Name').fill('claud');
+		await page.locator('#userEmail').fill('claud@gmail.com');
+		await page.locator('#currentAddress').fill('');
```
Esta forma de intentar dejar el campo vacío, es algo que un QA Manual haría probablemente, pero en el código tenemos que ser eficientes con sentido común, y no literalmente usar un método de fill cuando realmente en la vida real, nosotros como usuario no insertamos Ni el alma.

E-Gregorio replied:
He cerrado el PR anterior tal como me solicitaste y ya he creado uno nuevo. Este nuevo PR incluye todos los últimos cambios  realizado y está completamente alineado con las pruebas de QA. Las pruebas ya se ejecutaron y pasaron exitosamente, creando una variable de captura de datos JSON, usandolo como metodo de prueba, eliminando informacion de campos vacios. Se ajusto los test cases, eviatndo el uso innecesario de metodos "fill", en campos vacios. Gracias por la observacion, el nuevo PR esta listo para tu revision.

saiotest reviewed (changes requested):
Cierra este PR y crea uno nuevo, con todos los últimos cambios que me puedan confirmar que está todo al día con QA y no va a haber ningun problema. 
Por favor CIERRA vos mismo este PR.

Commit:
chore: Actualizacion del Archivo JSON.
test: Se corrigio la logica de los casos de pruebas, ajustando el manejo de campos vacios.
Buenas tardes
Se cerro el PR anterior tal como me lo solicitaste el dia Viernes 23/08/2024, creando uno nuevo con todo los cambios realizados.
Todos los test se ejecutaron y pasaron exitosamente. Tambien realice ajuste en la logica para evitar el uso de metodos "fill" en campos vacios.
Gracias por la observacion. El PR nuevo, esta listo para turevision y comentario.
